src/test/java/com/conveyal/gtfs/dto/StopTimeDTO.java

```diff
@@ -14,12 +15,25 @@
     public int continuous_pickup;
     public int continuous_drop_off;
 
+    // Additional GTFS Flex booking rule fields.
+    public String pickup_booking_rule_id;
+    public String drop_off_booking_rule_id;
+
+    // Additional GTFS Flex location groups and locations fields
+    public Integer start_pickup_dropoff_window;
+    public Integer end_pickup_dropoff_window;
```

Comment:
These should be time or string.

Reply:
Not here. They are defined as TimeField in Table.java, but here they should be ints matching other DTOs.